tests/test_validators.py

```diff
@@ -170,6 +176,12 @@ def test_validate_prefix_overlap():
     ("UNKNOWN::ASFOO", False),
     ("AS-FOO RIPE:AS-FOO", False),
     ("AS-FOO AS-FOO@RIPE", False),
+    ("RIPE::AS12345", False),
+    ("RIPE::RS15562:RS-FOO", False),
+    ("RIPE::AS12356:RS-FOO", False),
```

Comment:
however, `RIPE::AS123456:RS-FOO:AS-FOO` is not allowed, you can not mix and match `AS-` and `RS-` in hierarchical systme